perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/FacilitiesManagerMethod.java

```diff
@@ -278,12 +278,27 @@ public Void call(ApiCaller ac, Deserializer parms) throws PerunException {
 	 * @param facility Facility JSON object
 	 * @return Facility Created Facility object
 	 */
+	/*#
+	 * Creates a facility.
```

Comment:
You might want to note, that caller is made facility manager (same like its for createVo).